src/sentry/static/sentry/app/views/issueList/actions.jsx

```diff
@@ -562,12 +563,30 @@ const IssueListActions = createReactClass({
               </GraphToggle>
             </GraphHeader>
           </GraphHeaderWrapper>
-          <EventsOrUsersLabel className="align-right">
-            <ToolbarHeader>{t('Events')}</ToolbarHeader>
-          </EventsOrUsersLabel>
-          <EventsOrUsersLabel className="align-right">
-            <ToolbarHeader>{t('Users')}</ToolbarHeader>
-          </EventsOrUsersLabel>
+          <ToolbarHeader>
+            <EventsOrUsersLabel className="align-right">
```

Comment:
Can we "extend" `ToolbarHeader` here instead of nesting?

We can say `EventsOrUsersLabel` extends `ToolbarHeader` by:

```
const EventsOrUsersLabel = styled(ToolbarHeader)`
`
```

Reply:
Do we need `align-right` still?

Reply:
On wider screens align-right is still needed.